a7/__init__.py

```diff
@@ -0,0 +1,24 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import sys
+
+from .validate import validate_py3
+
+
+def main():
+    """Run validate on a single python file"""
+    if len(sys.argv) != 2:
```

Comment:
We shouldn't do this. There may very well be other things this package does so let's avoid extra Agent changes in future and make an arg before to specify action.